docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -1,58 +1,88 @@
+    guard groupStatus == SDLPermissionGroupStatus.allowed else { 
+        // Your app does not have permission to send the `DialNumber` request for its current HMI level
+        return 
     }
 
-    <#If making phone calls is supported, send the `DialNumber` RPC#>
-}];
+    // Your app has permission to send the `DialNumber` request for its current HMI level
+})
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+// TODO
+```
+!@
+
+## Checking if the Module Supports Calling a Phone Number
+Since making a phone call is a newer feature, there is a possibility that some legacy modules will reject your request because the module does not support the @![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ request. Once you have successfully connected to the module and you know that your SDL app has been opened, you can check the module's capabilities via the @![iOS]`SDLManager.systemCapabilityManager`!@@![android, javaSE, javaEE]`sdlManager.getSystemCapabilityManager`!@ as shown in the example below. Please note that you only need to check once if the module supports calling a phone number, however you must wait to perform this check until you know that the SDL app has been opened (i.e. the `hmiLevel` is non-`NONE`). 
+
+If you discover that the module does not support calling a phone number or that your app does not have the right permissions, you should disable any buttons, voice commands, menu items, etc. in your app that would send the @![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ request.
+
+@![iOS]
+##### Objective-C
+```objc
+// TODO
 ```
 
 ##### Swift
 ```swift
-sdlManager.systemCapabilityManager.updateCapabilityType(.phoneCall) { (error, systemCapabilityManager) in
-    var isDialNumberSupported = false
-    if error == nil {
-        isDialNumberSupported = systemCapabilityManager.phoneCapability?.dialNumberEnabled?.boolValue ?? false;
-    } else {
-        isDialNumberSupported = systemCapabilityManager.hmiCapabilities?.phoneCall?.boolValue ?? false
+func isDialNumberSupported(handler: @escaping (_ success: Bool, _ error: Error?) -> ()) {
+    // Check if the module has phone capabilities
+    guard (sdlManager.systemCapabilityManager.isCapabilitySupported(type: .phoneCall)) else {
+        return handler(false, nil)
+    }
+
+    // Legacy modules (pre RPC Spec v4.5) do not support the phone call capability so for versions less than 4.5 we will assume `DialNumber` is supported
+    guard let sdlMsgVersion = sdlManager.registerResponse?.sdlMsgVersion, SDLVersion(sdlMsgVersion: sdlMsgVersion).isGreaterThanOrEqual(to: SDLVersion(major: 4, minor: 5, patch: 0)) else {
+        return handler(true, nil)
     }
 
-    <#If making phone calls is supported, send the `DialNumber` RPC#>
+    // Check if the module supports the `DialNumber` request
+    sdlManager.systemCapabilityManager.updateCapabilityType(.phoneCall) { (error, systemCapabilityManager) in
```

Comment:
This should check the cached `phoneCapability` before calling `updateCapabilityType`

Reply:
Added.

Reply:
It doesn't appear that it was

Reply:
Sorry. I forgot to push. It should be fixed now.